From 6f4025eaea1446efd850e1acf3481100791d8811 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Thu, 1 Feb 2024 15:46:31 +0100
Subject: [PATCH 486/841] feat(efi): add metadata GUID define
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add a define to be able to use the metadata GUID in code.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git c21b27e3d99111f92f97ad751ab1c9bb3f289d35 feat(efi): add metadata GUID define]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I8add4caa23decf5c1c2f2ad322fc956a703c2d33
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/356465
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 include/drivers/partition/efi.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/drivers/partition/efi.h b/include/drivers/partition/efi.h
index 96c285737..59dea5701 100644
--- a/include/drivers/partition/efi.h
+++ b/include/drivers/partition/efi.h
@@ -31,6 +31,10 @@ static inline void *guidcpy(void *dst, const void *src)
 	  (c) & 0xffffU,			\
 	  { (d0), (d1), (d2), (d3), (d4), (d5), (d6), (d7) } }
 
+#define FWU_METADATA_GUID \
+	EFI_GUID(0x8A7A84A0U, 0x8387U, 0x40F6U, \
+		 0xABU, 0x41U, 0xA8U, 0xB9U, 0xA5U, 0xA6U, 0x0DU, 0x23U)
+
 #define NULL_GUID \
 	EFI_GUID(0x00000000U, 0x0000U, 0x0000U, 0x00U, 0x00U, \
 		 0x00U, 0x00U, 0x00U, 0x00U, 0x00U, 0x00U)
-- 
2.47.2

